### ISSUE TITLE
Map Marker Checkbox for Objects keeps duplicating Markers

### ISSUE THREAD

**Issue by Phenosi-PLUS** [R3voA3/3den-Enhanced#291]
### Description

When using the Map Marker checkbox for an placed object it tends to duplicate the creation of said markes for each time you start the mission or preview it.

### Steps To Reproduce

1. Place object
2. enable the 'Map Marker' Checkbox
3. It creates 1 marker as intended
4. start and stop preview
5. it creates another marker.

### Expected behavior

It will eventually create so many duplicates of the markers that it reaches the total marker count of 9999. this breaks the mission: It lags the editor severly, it cannot be loaded in any MP enviroment anymore.

### 3den Enhanced Version

7.8.4

### Arma 3 Version

2.14

### Is the issue reproduceable?

- [X] Always
- [X] Sometimes
- [ ] Only once

### Game breaking?

- [X] Yes, it is
- [ ] No, it isn't

**Comment by R3voA3** [comment]
Thanks for the report. 

I am not sure however what attribute you are talking about. Mind uploading a screenshot?

**Comment by Phenosi-PLUS** [comment]
My apologies for for the report, this option was made by O&T 3den Expansion mod.
my mistake.

**Comment by R3voA3** [comment]
No problem. =)
